integration/access_log_json_test.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"time"
+
+	"bufio"
+	"github.com/go-check/check"
+	checker "github.com/vdemeester/shakers"
+)
+
+// AccessLogSuite
+type AccessLogJSONSuite struct{ BaseSuite }
+
+func (s *AccessLogJSONSuite) TestAccessLogJSON(c *check.C) {
+	// Ensure working directory is clean
+	os.Remove("access-log.json")
+	os.Remove("traefik.log")
+
+	// Start Traefik
+	cmd := exec.Command(traefikBinary, "--configFile=fixtures/access_log_json_config.toml")
+	err := cmd.Start()
+	c.Assert(err, checker.IsNil)
+	defer cmd.Process.Kill()
+	defer os.Remove("access-log.json")
+	defer os.Remove("traefik.log")
+
+	time.Sleep(500 * time.Millisecond)
```

Comment:
You're also asking me to upgrade existing code (access_log_test.go), which is related.